CONTRIBUTING.md

```diff
@@ -9,6 +9,100 @@
 5. Push to the branch (`git push origin my-new-feature`)
 6. Create a new Pull Request
 
+### Building
+
+A gradle wrapper is included so these tasks can run without any prior installation of gradle. The Linux/OSX form of the commands, given
+below, is:
+
+```bash
+./gradlew <task name>
+```
+
+For Windows, use the batch file:
+
+```cmd
+gradlew.bat <task name>
+```
+
+To build the library’s AAR package, execute:
+
+```bash
+./gradlew chat-android:assemble
+```
```

Comment:
The built AAR package will be available in the `chat-android/build/outputs/aar/` directory.

Reply:
fixed